.github/workflows/release.yaml

```diff
@@ -135,8 +135,11 @@ jobs:
       with:
         ref: ${{ needs.create-release-tag.outputs.develop_name }}
         
-    - name: Build images
-      run: ./cd/build_images.sh
+    - name: Build Accumulo 2 images
+      run: ./cd/build_images.sh ./docker/accumulo2.env
+
+    - name: Build Accumulo 1 images
+      run: ./cd/build_images.sh ./docker/accumulo1.env
```

Comment:
This builds all images _using_ the given version of Accumulo. Could this be changed to avoid confusion that this script only builds Accumulo images.